contrib/hadoop-store-builder/src/java/voldemort/store/readonly/mr/HadoopStoreBuilder.java

```diff
@@ -220,7 +220,11 @@ public void build() {
             FileSystem tempFs = tempDir.getFileSystem(conf);
             tempFs.delete(tempDir, true);
 
-            long size = sizeOfPath(tempFs, inputPath);
+            //use a list in case we pass a csv path list
```

Comment:
Please look at : https://aws.amazon.com/blogs/aws/amazon-s3-performance-tips-tricks-seattle-hiring-event/
amazon recommend hash the s3 directory for large inputs. s3 is not really a directory structure but a key/values store.
In this case, we need to create an array contains all the input files instead of the root of path.